.changeset/big-paws-shave.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/stylelint-polaris': major
```

Comment:
Question, if we are only removing rules, would this be a breaking change?

I think we could leave this as a `minor`

Reply:
> 🤔 I feel like I vaguely remember that after thinking it over @aaronccasanova might have discovered that it could cause breaking changes for consumers. @aaronccasanova do you remember?

Yes, I can think of two CI failures off the top: [needless disables](https://stylelint.io/user-guide/options/#reportneedlessdisables) and [invalid scopes](https://stylelint.io/user-guide/options/#reportinvalidscopedisables).

`major` makes sense to me. @lgriffee do we need to update the [version matchups](https://polaris.shopify.com/tools/stylelint-polaris#version-matchups) in this PR?